src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  console.log(todos);
+
+  return (
+    <>
+      {todos.map((todo: Todo) => (
+        <div key={todo.id} data-cy="Todo" className="todo completed">
```

Comment:
make component for todo item.
All logic which related only for todo item should be in TodoItem component.